odl/test/tomo/geometry/geometry_test.py

```diff
@@ -665,38 +593,111 @@ def test_helical_cone_flat_props(shift):
                              [0, 0, 1]])
 
     # pitch_offset
-    geom = odl.tomo.HelicalConeFlatGeometry(apart, dpart, src_rad, det_rad,
-                                            pitch, pitch_offset=0.5)
+    geom = odl.tomo.ConeFlatGeometry(apart, dpart, src_rad, det_rad,
+                                     pitch=pitch, pitch_offset=0.5)
```

Comment:
A thought on the naming here. `pitch_offset` is perhaps not the best name, since at least for me this implies that the angles will change. We can maybe rename this to `axis_offest`, since it is an offset along the axis?

I think finalizing the breaking changes once and for all here would be a good idea.